[connector/spanmetrics] fix(docs): add example of R.E.D metrics for span data

karmingc opened the pull request:
**Description:** This is to clarify the metrics name generated.

**Link to tracking Issue:** [26520](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/26520)

**Testing:** n/a docs

**Documentation:** n/a

karmingc committed:
fix(docs): add example of R.E.D metrics for span data
This is to clarify the metrics name generated.

albertteoh reviewed (approved):
Thanks!

albertteoh commented on connector/spanmetricsconnector/README.md:
````diff
@@ -31,11 +31,23 @@ Aggregates Request, Error and Duration (R.E.D) OpenTelemetry metrics from span d
 relevant duration histogram time bucket for each unique set dimensions.
 
+```
+duration{http.method="GET",http.route="/shipping/{shippingId}"}
````
To be more accurate, shouldn't every example contain `service.name`, `span.name`, `span.kind` and `status.code`? See next paragraph.

karmingc committed:
fix(docs): add the default attributes in the example
resolves: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29115#discussion_r1392459119
